src/main/java/com/duke/task/Event.java

```diff
@@ -0,0 +1,35 @@
+package com.duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents an Event Task.
+ */
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Creates a event task with the given description and date information.
+     *
+     * @param description description of the event.
+     * @param a           the date information of the event.
+     * @throws DateTimeParseException when the date information is in invalid format.
+     */
+    public Event(String description, String a) throws DateTimeParseException {
```

Comment:
The parameter name "a" again provides no intuitive understanding of the parameter or what it does